src/main/java/com/google/sps/servlets/CheckPermissionsServlet.java

```diff
@@ -15,77 +14,67 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
-// import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
-// import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
-// import com.google.api.client.http.HttpTransport;
-// import com.google.api.client.json.JsonFactory;
-// import com.google.api.client.json.jackson2.JacksonFactory;
-// import com.google.api.services.iam.v1.iam;
-// import com.google.api.services.iam.v1.model.ListServiceAccountsResponse;
-// import com.google.api.services.iam.v1.model.ServiceAccount;
-// import java.io.IOException;
-// import java.security.GeneralSecurityException;
-// import java.util.Arrays;
-
+import com.google.gson.Gson;
+import com.google.common.collect.Lists;
+import java.io.FileInputStream;
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.iam.v1.Iam;
-import com.google.api.services.iam.v1.IamScopes;
-import com.google.api.services.iam.v1.model.ListServiceAccountsResponse;
-import com.google.api.services.iam.v1.model.ServiceAccount;
+import com.google.api.services.iam.v1.model.Policy;
 import com.google.auth.http.HttpCredentialsAdapter;
 import com.google.auth.oauth2.GoogleCredentials;
-import java.io.IOException;
 import java.security.GeneralSecurityException;
 import java.util.Collections;
 import java.util.List;
-import com.google.gson.Gson;
 
 @WebServlet("/check-permissions")
 public class CheckPermissionsServlet extends HttpServlet {
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String projectId = request.getParameter("projID");
-
+    String jsonPath = projectId + "-key.json";
+    String resource = "projects/" + projectId + "/serviceAccounts/106253050877809441203";
```

Comment:
I think it would be better not to hard-code service-accounts IDs, but to pass them as parameters to the class Constructor

Reply:
Good point! That's now changed in e7f992e :)